### PR TITLE
Add example and tests for TS -> Proto -> Kotlin path

### PR DESCRIPTION
**Pull request by piotrswigon**

### CODE REVIEW

File: src/tools/tests/test-data/Example.arcs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+external particle Reader
+  data: reads Thing {name: Text}
+
+recipe PassThrough
+  thing: create
+  Writer
+    data: writes thing
+  Reader
+    data: reads thing
```
Comment (csilvestrini) [inline src/tools/tests/test-data/Example.arcs]: Nit: newline at EOF

File: src/tools/manifest2proto.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import protobuf from 'protobufjs';
+
+const rootNamespace = protobuf.loadSync('./java/arcs/core/data/manifest.proto');
+const manifestProto = rootNamespace.lookupType('arcs.Manifest');
+
+export async function serialize2proto(path: string): Promise<Uint8Array> {
+  const manifest: Manifest = await Runtime.parseFile(path);
+
+  if (manifest.errors.some(e => e.severity === ErrorSeverity.Error)) {
+    throw new Error(`Problems found in manifest '${path}'`);
```
Comment (csilvestrini) [inline src/tools/manifest2proto.ts]: Can you include the errors in the error message?

Comment (piotrswigon) [reply src/tools/manifest2proto.ts]: This whole thing is actually not needed, Runtime.parseFile will throw for errors. I've removed this.

File: src/tools/tests/test-data/Example.arcs (hunk 1 above)
Comment (piotrswigon) [reply src/tools/tests/test-data/Example.arcs]: done!

**Pull request merged**